frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -9,56 +9,46 @@ import { WINTOOLS_CONTAINER_NAMES } from '../modals/cdrom-vm-modal/constants';
 import { VMKind } from '../../types';
 import { V1Disk } from '../../types/vm/disk/V1Disk';
 
-const DiskSummaryRow: React.FC<DiskSummaryRowProps> = ({ title, value }) => (
-  <div className="kubevirt-disk-summary__disk">
-    <div id={`kubevirt-disk-summary-disk-title-${title}`}>{title}</div>
-    <div id={`kubevirt-disk-summary-disk-value-${value}`}>{value}</div>
-  </div>
-);
+import './disk-summary.scss';
 
 export const DiskSummary: React.FC<DiskSummaryProps> = ({ disks, vm }) => (
-  <div className="kubevirt-disk-summary">
-    {disks.map(({ name }, i) => {
+  <dl className="oc-vm-details__datalist kubevirt-disk-summary">
+    {disks.map(({ name }) => {
       const container = getContainerImageByDisk(vm, name);
       const pvc = getPVCSourceByDisk(vm, name);
       const url = getURLSourceByDisk(vm, name);
+      const nameKey = `kubevirt-disk-summary-disk-title-${name}`;
+      let value = '';
 
       if (_.includes(WINTOOLS_CONTAINER_NAMES, container)) {
-        return (
-          <DiskSummaryRow
-            key={`disk-summary-${name}`}
-            title={`Drive ${i + 1}: Windows Tools`}
-            value={container}
-          />
-        );
-      }
-      if (container) {
-        return (
-          <DiskSummaryRow
-            key={`disk-summary-${name}`}
-            title={`Drive ${i + 1}: Container`}
-            value={container}
-          />
-        );
-      }
-      if (url) {
-        return (
-          <DiskSummaryRow key={`disk-summary-${name}`} title={`Drive ${i + 1}: URL`} value={url} />
-        );
+        value = `Windows Tools: ${container}`;
+      } else if (container) {
+        value = `Container: ${container}`;
+      } else if (url) {
+        value = `URL: ${url}`;
+      } else if (pvc) {
+        value = `PVC: ${pvc}`;
       }
+
       return (
-        <DiskSummaryRow key={`disk-summary-${name}`} title={`Drive ${i + 1}: PVC`} value={pvc} />
+        <>
+          <dt id={nameKey} key={nameKey}>
+            {name}
+          </dt>
+          <dd
+            id={`${nameKey}-info`}
+            key={`${nameKey}-info`}
+            className="co-vm-details-cdroms__datalist-dd text-secondary"
```

Comment:
I think you can drop the .text-secondary class here.

Reply:
hmm ... when looking at the  ( bigger :-) ) picture below,  the "cd-drive-2" ( 3rd header ? ) looks very similar to the "IP Adress" ( 2nd title ? ), IMHO making the text darker will make this line look more important than the higher order headers, WDYT ?

![hello · Details · OKD(7)](https://user-images.githubusercontent.com/2181522/70242527-a1210980-1779-11ea-881e-05e0dffedf7d.png)

Reply:
Isn't the `.text-secondary` class on the `container: path/to/container` text not on the `cd-drive-1` text?
Can we also just have this like
```
Drive 1
PVC:hello-disk1
```
Makes it look a bit nicer.

Reply:
perhaps using var(--pf-global--Color--400) for the sub-headings (Drive 1) would be help set them apart a bit more from the secondary headings.

Reply:
+1 make sense to me too, I will do follow-up PR for tweaking css, and rename it :-)